apps/verticalhome/style/css/app.css

```diff
@@ -15,9 +15,10 @@ body {
 
 gaia-grid {
   display: block;
-  height: 100%;
+  /* Minus the bottom margin and the search-box height */
+  min-height: calc(100% - 5rem - 5rem);
```

Comment:
Yes - the height does nothing as we set this manually in grid_view, but if we don't set min-height, the overflow: hidden means we end up clipping items that get dragged below the container (which is also the reason for changing margin-bottom to padding-bottom)